src/types/targetValidators.ts

```diff
@@ -139,6 +140,18 @@ export const validateAuthConfig = (authConfig: AuthConfig): AuthConfig => {
   return authConfig
 }
 
+export const validateAuthConfigForSas9 = (
```

Comment:
please rename it to `validateAuthConfigSAS9 `

Reply:
and you should also validate that `userName` and `password` are not empty strings